cortex/blender/__init__.py

```diff
@@ -225,8 +284,24 @@ def fs_cut(fname, subject, hemi, freesurfer_subject_dir=None, blender_path=defau
         """.format(tfname=tf.name)
         _call_blender(fname, code, blender_path=blender_path)
 
-def write_patch(bname, pname, mesh="hemi", blender_path=default_blender):
```

Comment:
Hm, I understand that you are generalizing and splitting this function, but as this is an exported function, it seems like this could break existing workflows. Maybe keep "write_patch" with a FutureWarning, that points to the default behavior.

Some users might be manually using these functions.